salt/states/pkgrepo.py

```diff
@@ -344,7 +344,7 @@ def managed(name, ppa=None, **kwargs):
             if disabled is not None \
             else salt.utils.is_true(enabled)
 
-    elif os_family == 'nilinuxrt':
+    elif os_family in ('nilinuxrt', 'poky'):
```

Comment:
It is "NILinuxRT" or "lilinuxrt"?

Reply:
Previously in that code (not by me) the os_family variable is "normalized" with .lower().

Reply:
Could you please fix this? Such "normalisation" usually leads to a nasty bugs than later we have all over the place.

Reply:
So, it seems that the .lower() is originally from e7fb3095ceeecc1e5507754515a34e2d498a6e75. Was there a reason for it then?